### DISCUSSION TITLE
Proposal: integrate `macos_window_utils` into the `macos_ui` organization

### DISCUSSION THREAD

**Discussion by GroovinChip** [Adrian-Samoticha/macos_window_utils.dart#19] [Ideas]
Hi, Adrian!

Your hard work with `macos_window_utils` is incredible, and is quickly becoming an integral part of Flutter for macOS. We are already [integrating your plugin with mine](https://github.com/macosui/macos_ui/pull/377), so I would like to invite you to become a member of the recently created [macos_ui GitHub organization](https://github.com/macosui), and migrate your repository to it. 

**You would retain ownership and complete control of your project.**

I believe that joining forces in this way would allow us to better coordinate and collaborate our efforts to improve Flutter for macOS. Given the pending integration of our works and my plans for growing the macos_ui "ecosystem", I think it would be greatly beneficial for us both, as well as for the Flutter community at large.

Please let me know your thoughts! No pressure, of course.

Cheers,

Reuben (GroovinChip)

**Comment by Adrian-Samoticha** [comment]
I assume this would also mean that I'd be able to create new repositories inside the organization as well? I've been thinking about splitting some functionality (such as #15) into stand-alone repositories if they are usable without touching the application's `NSWindow`, and then depending on those from within macos_window_utils. Those repos would need to be located inside the macos_ui organization as well.

**Comment by GroovinChip** [comment]
> I assume this would also mean that I'd be able to create new repositories inside the organization as well? I've been thinking about splitting some functionality (such as #15) into stand-alone repositories if they are usable without touching the application's `NSWindow`, and then depending on those from within macos_window_utils. Those repos would need to be located inside the macos_ui organization as well.

Yes, as a member you'd be able to create new repositories within the organization :)

**Reply by Adrian-Samoticha** [reply]
Sound good. In that case, I'm open to doing so.

**Reply by GroovinChip** [reply]
Excellent! Do you think we should do it before or after the integration?

**Reply by Adrian-Samoticha** [reply]
I think it'd be preferable to do so pre-integration. If I do end up splitting #15 into a separate package, it'd be simpler to create its repo in the organization as opposed to migrating it later.

**Reply by GroovinChip** [reply]
> I think it'd be preferable to do so pre-integration. If I do end up splitting #15 into a separate package, it'd be simpler to create its repo in the organization as opposed to migrating it later.

Great! I'll add you to the org. Once I do, feel free to migrate the repo at your convenience.
